www/js/app.js

```diff
@@ -497,17 +565,17 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                     else resolve(cache);
                 };
                 // Ask Service Worker for its cache status and asset count
-                navigator.serviceWorker.controller.postMessage({
+                if (navigator.serviceWorker.controller) navigator.serviceWorker.controller.postMessage({
```

Comment:
Is this test necessary, as it already has been tested a few lines above?

Reply:
I'll check, it could be left over from a previous iteration of the code.

Reply:
Removed.